goreleaser/README.md

```diff
@@ -8,6 +8,17 @@ tagline: |
 To update or switch versions, run `webi goreleaser@stable` (or `@v0.174`,
 `@beta`, etc).
 
```

Comment:
This is correct, but you can drop this commit because I have another PR open that includes it:
- https://github.com/webinstall/webi-installers/pull/715

Reply:
Alright, dropped.